### PR TITLE
translate s2ibt description.

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: zhuxiaoyang <sunzhu@yunify.com>

### CODE REVIEW

**Comment by runzexia** [comment]
I recommend multi-language support through annotations so that we can avoid frequent modification of field definitions.

**Comment by soulseen** [comment]
> I recommend multi-language support through annotations so that we can avoid frequent modification of field definitions.

good point

File: config/templates/tomcat.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ metadata:
     builder-type.kubesphere.io/b2i: "b2i"
     binary-type.kubesphere.io: "war"
   annotations:
+    descirptionCN: "Tomcat应用的S2i构建器模版，通过该模版可构建出直接运行的应用镜像。"
```
Comment (runzexia) [inline config/templates/tomcat.yaml]: Add space on sides of the term

File: config/templates/python.yaml [hunk 2]
```diff
@@ -5,6 +5,8 @@ metadata:
     controller-tools.k8s.io: "1.0"
     builder-type.kubesphere.io/s2i: "s2i"
   annotations:
+    descirptionCN: "Python应用的S2i构建器模版。Python是一种易于学习，功能强大的编程语言。 它具有高效的高级数据结构和简单但有效的面向对象编程方法。"
```
Comment (runzexia) [inline config/templates/python.yaml]: Add space on sides of the term

File: config/templates/nodejs.yaml [hunk 3]
```diff
@@ -5,6 +5,8 @@ metadata:
     controller-tools.k8s.io: "1.0"
     builder-type.kubesphere.io/s2i: "s2i"
   annotations:
+    descriptionEN: "Node.js available as container is a base platform for building and running various Node.js applications and frameworks. Node.js is a platform built on Chrome's JavaScript runtime for easily building fast, scalable network applications. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient, perfect for data-intensive real-time applications that run across distributed devices."
+    descirptionCN: "Nodejs应用的S2i构建器模版。Node.js是基于Chrome的JavaScript运行时构建的平台，可轻松构建快速，可扩展的网络应用程序。"
```
Comment (runzexia) [inline config/templates/nodejs.yaml]: Add space on sides of the term

File: config/templates/java.yaml [hunk 4]
```diff
@@ -7,6 +7,9 @@ metadata:
     builder-type.kubesphere.io/b2i: "b2i"
     binary-type.kubesphere.io: "jar"
   annotations:
+    descriptionCN: "Java应用的S2i构建器模版，并且集成了Jolokia代理。通过该模版可构建出直接运行的应用镜像。"
```
Comment (runzexia) [inline config/templates/java.yaml]: Add space on sides of the term

File: config/templates/binary.yaml [hunk 5]
```diff
@@ -6,6 +6,8 @@ metadata:
     builder-type.kubesphere.io/b2i: "b2i"
     binary-type.kubesphere.io: "binary"
   annotations:
+    descriptionEN: "This is a B2i builder template for binary build"
+    descriptionCN: "二进制文件的B2i构建器模版"
```
Comment (runzexia) [inline config/templates/binary.yaml]: Add space on sides of the term

**Pull request merged**